tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -156,15 +156,15 @@ exports[`test hug.js 1`] = `
   {__DEV__
     ? this.renderDevApp()
     : <div>
-      {routes.map(route => (
-        <MatchAsync
-          key={\`\${route.to}-async\`}
-          pattern={route.to}
-          exactly={route.to === \"/\"}
-          getComponent={routeES6Modules[route.value]}
-        />
-      ))}
-    </div>}
+        {routes.map(route => (
+          <MatchAsync
+            key={\`\${route.to}-async\`}
+            pattern={route.to}
+            exactly={route.to === \"/\"}
+            getComponent={routeES6Modules[route.value]}
+          />
+        ))}
+      </div>}
```

Comment:
Oops, looks like this just got merged anyway! Presumably the issue I highlighted above will get brought up as an issue later if people don't like it, and can be discussed there.